contracts/cw1-subkeys/src/multitest/admin_vote_contracts.rs

```diff
@@ -0,0 +1,295 @@
+use anyhow::{bail, Result as AnyResult};
```

Comment:
Why such a chuge contract in the multitest? Can't you minimize it? Or extract as separate contract?